AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/PackageManifestBuilder.swift

```diff
@@ -10,8 +10,12 @@ import AWSCLIUtils
 
 /// Builds the contents of the package manifest file.
 struct PackageManifestBuilder {
+
     struct Service {
-        let name: String
+        let moduleName: String
+        let codegenName: String
+        let dependencies: [String]
+        let isInternal: Bool
     }
```

Comment:
An array of this data structure is passed in, with info about every service client to be included in the manifest.